### PR TITLE
Trademark updates

### LINKED ISSUE #4221: MicroMasters® trademark updates
edX is asking all their partners to update their electronic communication to better protect the MicroMasters trademark. The MITx MicroMasters team has asked us to make the following changes to the portal:

mm site_homepage_screenshot 1
mm site_homepage_screenshot 2
mm site_homepage_screenshot 3
mm site_homepage_screenshot 4
mm site_homepage_screenshot 5

In general, the requirements is that the first mention of "MicroMasters" on the page is followed by a ® mark, and that MicroMasters never be used standalone, but only to modify a noun, like Program or Courses. 

We are getting in touch with the various course teams about the CMS content.

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #4221

#### What's this PR do?
Introducing updates described in the issue.
I had to create two new svg files for the header. The font is not identical, but that can be updated if we find the original font.

#### How should this be manually tested?
Make sure all the changes are there

screen shot 2019-01-09 at 4 47 50 pm
screen shot 2019-01-09 at 4 48 01 pm

### CODE REVIEW

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -80,16 +80,16 @@ <h2>
...
             <div class="youtubevideowrap">
               <div class="video-container">
                 <iframe width="560" height="315" src="https://www.youtube.com/embed/7lVvOSJpFGY" frameborder="0" allowfullscreen></iframe>
               </div>
             </div>
-              <p>MicroMasters is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MITx graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
-              <p>MicroMasters courses offer the same learning and challenge as MIT courses.</p>
+              <p>A MicroMasters program credential is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MITx graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters program credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]:
```suggestion
              <p>A MicroMasters program credential is a professional and academic credential for online learners from anywhere in the world. Learners who pass an integrated set of MIT’s graduate-level courses on edX.org, and one or more proctored exams, will earn a MicroMasters program credential from MITx, and can then apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
```

Feel free to break this into multiple lines while you're in here, if you like.

File: static/scss/homepage.scss [hunk 2]
```diff
@@ -52,8 +52,8 @@ body.app-media {
 
       .micromasters-logo {
         height: 23px;
-        width: 152px;
-        top: 10px;
+        width: 252px;
+        top: 8px;
```
Comment (pdpinch) [inline static/scss/homepage.scss]: Just curious: why this change?

File: static/scss/homepage.scss [hunk 3]
```diff
@@ -274,6 +274,11 @@ body.app-media {
         }
       }
 
+      span {
+        vertical-align: super;
+        font-size: 14px;
+      }
+
```
Comment (pdpinch) [inline static/scss/homepage.scss]: nice touch

File: ui/templates/background-images.css [hunk 4]
```diff
@@ -9,10 +9,10 @@
   background-image: url({% static "images/mit_logo_on.svg" %});
 }
 .navbar .navbar-header .micromasters-logo {
-  background-image: url({% static "images/micromasters_logo.svg" %});
+  background-image: url({% static "images/micromasters_logo_programs.svg" %});
 }
 .navbar .navbar-header .micromasters-logo:hover {
-  background-image: url({% static "images/micromasters_logo_on.svg" %});
+  background-image: url({% static "images/micromasters_logo_on_programs.svg" %});
```
Comment (pdpinch) [inline ui/templates/background-images.css]: I don't think we need a special version just for this page. We should change the "logo" everywhere

Comment (annagav) [reply ui/templates/background-images.css]: This particular logo is used only in this header on the homepage.

File: static/scss/homepage.scss (hunk 2 above)
Comment (annagav) [reply static/scss/homepage.scss]: The width of the logo is different now, so positioning had to adjusted.

File: cms/templates/cms/home_page.html [hunk 5]
```diff
@@ -80,16 +80,16 @@ <h2>
         <div class="col-md-12 ">
           <section class="sub-banner-section">
             <div class="sub-banner-text">
-            <h3>What is the MicroMasters Credential?</h3>
+              <h3>What is the MicroMasters<span>&#174;</span> Program Credential?</h3>
```
[the remark this answers is not in the document]
Comment (annagav) [reply cms/templates/cms/home_page.html]: Whatever I found first.

File: ui/templates/background-images.css (hunk 4 above)
Comment (pdpinch) [reply ui/templates/background-images.css]: Hmm. I have no idea why we do that, but I don't think I want to open than can of worms.

**Comment by pdpinch** [comment]
You can go ahead and merge this after you make the change in https://github.com/mitodl/micromasters/pull/4222#discussion_r246792066
